spec/views/people/new.html.erb_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe "people/new", type: :view do
+  before(:each) do
+    assign(:person, Person.new(
+      :bio => "MyText",
```

Comment:
Indent the first parameter one step more than `Person.new(`.<br>Use the new Ruby 1.9 hash syntax.